examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from merlin.models.data.movielens import get_movielens
</pre><p>This might be a little out of bounds from your team tech writer, but could the package be "merlin.models.demo" or "m.m.d.movielens"?  That way, it's clear this is for demo purposes and production classes and functions can exist in <code>merlin.models.data</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I will create a Feature Request, that we can discuss it with the engineering team, what is the best structure.</p><p>I dont think we have time to organize it for next release</p>